s3fs/core.py

```diff
@@ -685,7 +685,9 @@ async def _find(self, path, maxdepth=None, withdirs=None, detail=False, prefix="
     async def _mkdir(self, path, acl="", create_parents=True, **kwargs):
         path = self._strip_protocol(path).rstrip("/")
         bucket, key, _ = self.split_path(path)
-        if not key or (create_parents and not await self._exists(bucket)):
+        if await self._exists(bucket):
+            await self._ls(bucket)
```

Comment:
Why list here?

Reply:
my bad. I mean to leave it to do nothing when bucket already exist, will update.